### PR TITLE
Plutus Tools duplicate topics deleted

### PR DESCRIPTION
**Pull request by joseph-fajen**
These changes are for PLT-626: deleting pages in Plutus Tools RTD that are duplicated in Plutus Core RTD. One page needs some more thought and input: 

https://plutus-apps.readthedocs.io/en/latest/plutus/howtos/exporting-a-script.html
Can we replace this page with a link to more up-to-date content? 

If these deletions are ok with reviewers, we can merge and then the Plutus Tools RTD will no longer contain this duplicated content.

### CODE REVIEW

File: doc/plutus/troubleshooting.rst
Comment (sjoerdvisscher) [inline doc/plutus/troubleshooting.rst:67]: This seems the wrong way around. The Ledger and state machine errors should stay. The prelude errors should go.

File: doc/plutus/tutorials/basic-validators.rst
Comment (sjoerdvisscher) [inline doc/plutus/tutorials/basic-validators.rst:39]: `BasicValidators.hs` should be removed too.

File: doc/plutus/tutorials/basic-minting-policies.rst
Comment (sjoerdvisscher) [inline doc/plutus/tutorials/basic-minting-policies.rst:39]: `BasicPolicies.hs` should be removed too.

File: doc/plutus/tutorials/plutus-tx.rst
Comment (sjoerdvisscher) [inline doc/plutus/tutorials/plutus-tx.rst:89]: `BasicPlutusTx.hs` should be removed too.

**Review by sjoerdvisscher — CHANGES_REQUESTED** [review]

**Review by michaelpj — APPROVED** [review]

**Review by koslambrou — APPROVED** [review]

File: doc/plutus/troubleshooting.rst
Comment (joseph-fajen) [reply doc/plutus/troubleshooting.rst:67]: OK, I've made that change. Should the Currency Errors also stay?

File: doc/plutus/tutorials/basic-validators.rst
Comment (joseph-fajen) [reply doc/plutus/tutorials/basic-validators.rst:39]: Removed

File: doc/plutus/tutorials/basic-minting-policies.rst
Comment (joseph-fajen) [reply doc/plutus/tutorials/basic-minting-policies.rst:39]: Removed

File: doc/plutus/tutorials/plutus-tx.rst
Comment (joseph-fajen) [reply doc/plutus/tutorials/plutus-tx.rst:89]: Removed

File: doc/plutus/tutorials/basic-validators.rst
Comment (joseph-fajen) [reply doc/plutus/tutorials/basic-validators.rst:39]: Actually after I removed the file `BasicValidators.hs` I got some build errors, so I put it back. However, the file `basic-validators.rst` has been removed.

File: doc/plutus/tutorials/basic-minting-policies.rst
Comment (joseph-fajen) [reply doc/plutus/tutorials/basic-minting-policies.rst:39]: I've removed the file `basic-minting-policies.rst`, but just to make sure I don't get any build errors, I'm leaving the file `BasicPolicies.hs` because I don't think it causes any problems by remaining.

File: doc/plutus/tutorials/plutus-tx.rst
Comment (joseph-fajen) [reply doc/plutus/tutorials/plutus-tx.rst:89]: I've removed `plutus-tx.rst` so that the page is no longer in read the docs. But I've left the file `BasicPlutusTx.hs` because I don't think it causes any problems by being there.

File: doc/plutus/tutorials/basic-validators.rst
Comment (sjoerdvisscher) [reply doc/plutus/tutorials/basic-validators.rst:39]: Ah, `BasicValidators.hs` is also used by `doc/plutus/howtos/exporting-a-script.rst`.

File: doc/plutus/troubleshooting.rst
Comment (sjoerdvisscher) [reply doc/plutus/troubleshooting.rst]: Yes.

**Comment by sjoerdvisscher** [comment]
It looks like we need https://docs.readthedocs.io/en/stable/guides/intersphinx.html to link from plutus tools docs to plutus core docs...

**Review by raduom — APPROVED** [review]

File: doc/plutus/tutorials/basic-minting-policies.rst
Comment (koslambrou) [reply doc/plutus/tutorials/basic-minting-policies.rst:39]: I'd advice removing `BasicPolicies` if it's not needed. You can just run `cabal build plutus-doc` locally to test that it's not used anywhere.

File: doc/plutus/tutorials/plutus-tx.rst
Comment (koslambrou) [reply doc/plutus/tutorials/plutus-tx.rst:89]: Again, should be removed if it's not used.

**Review by koslambrou — APPROVED** [review]

**Comment by sjoerdvisscher** [comment]
> For which parts exactly? You mean that we could create links which point to stuff in plutus core RTD?

Yes. I was looking at the build failure: `undefined label: what_is_the_plutus_platform`. This page now only exists in the plutus core RTD.

**Review by sjoerdvisscher — APPROVED** [review]
Looks good!

To fix the build I added the option to link to plutus core documentation.
The build is still broken but that is an unrelated issue that all builds have at the moment.

**Comment by joseph-fajen** [comment]
@sjoerdvisscher Thanks for looking it over and approving. Given that the build is still broken, is there anything I need to do before this can merge?

**Comment by sjoerdvisscher** [comment]
Yes, we can't merge this yet. My change introduced a new problem. See https://input-output-rnd.slack.com/archives/C029B8E2RHA/p1662377259506569

**Comment by joseph-fajen** [comment]
@sjoerdvisscher It appears that some checks/builds are not completing. Do you know what needs to be fixed?

**Pull request merged**
